src/status_im/subs/activity_center.cljs

```diff
@@ -6,35 +6,33 @@
             [clojure.string :as string]))
 
 (re-frame/reg-sub
- :activity-center/notifications-read
+ :activity-center/notifications
```

Comment:
Sorry @flexsurfer, I didn't understand this one comment. This subscription looks like the following, what's your suggestion?

```clojure
(re-frame/reg-sub
 :activity-center/notifications
 (fn [db]
   (get-in db [:activity-center :notifications])))
```

Reply:
I think @flexsurfer means that we should have a subscription
```
(reg-root-key-sub :activity-center :activity-center)
 ```

and then compose the 3 subscriptions:
```
(re-frame/reg-sub
:activity-center/subscriptions
 :<- [:activity-center]
 (fn [activity-center]
   (:notifications activity-center)))
```

It's tempting to do:

```
```
(re-frame/reg-sub
:activity-center/subscriptions
 :<- [:activity-center]
  :notifications)
```
But it's prone to error as if a second parameter is passed and `activity-center` is nil it would return that one.

Reply:
```clojure
(reg-root-key-sub :activity-center :activity-center)


(re-frame/reg-sub
 :activity-center/notifications
 :<- [:activity-center]
 (fn [activity-center]
   (:notifications activity-center)))
```

Reply:
db should never be used in `re-frame/reg-sub`

Reply:
mhm for some reason i didn't have all your comments :(

Reply:
Okay, thanks. Now I'm curious to see why creating the `:activity-center` root subscription is better. I'm assuming it's for performance reasons, because layer-3 subs use cached data from layer-2.

Reply:
Done.

```
Refactor: use more layer-2 subs
```

https://github.com/status-im/status-mobile/pull/14125/commits/81e2f4bec31466191d3ee80c23fc64efbc21cf6a